test_suite/README.md

```diff
@@ -223,59 +204,57 @@ def test_something(peripheral):
 
 ## Interacting with devices under test (DUT)
 
-Boards are passed to the test using the fixture mechanism.
+Boards are passed to the test using the fixture mechanism.<!--devices, boards and targets seem to be used interchangeably-->
```

Comment:
Indeed, boards and devices are used interchangeably. Target should be used for family of boards like it is used in the section `Select targets`

Reply:
We don't use target to mean family - we use it to mean "A destination for a software build, for example an MCU, development board or custom board. Build targets are defined in either targets.json or custom_targets.json. A build (with Mbed CLI, the Online Compiler and Mbed Studio) is always done for a specific target."

And we try not to use device, only board, when talking about hardware.

Reply:
I think we share the same understanding of what a target is. Build targets can be used on any board that have the characteristics described in the target. That's what I meant by family of boards. Maybe I should have used `class of boards` but it is broad too.   

> And we try not to use device, only board, when talking about hardware.

That doesn't seems to be the case in green tea [documentation](https://os.mbed.com/docs/mbed-os/v6.3/debug-test/greentea-for-testing-applications.html) where device is used more often than board. The term `Device Under Test` is fairly common when we refer to device validation through testing. It would be nice to unify usage though.

Reply:
That's because Greentea predates the new vocab guidelines. The problem with Device is that Keil use it to mean something else.